test/color/setting_test.dart

```diff
@@ -180,6 +180,63 @@ void main() {
 
         await screenMatchesGolden(tester, 'color_setting_nostroke-example-1');
       });
+
+      processingLegacySpecTest('background in clear() replace background in setup() and drawn rect', (tester) async {
+        await tester.pumpWidget(
+          Processing(
+            sketch: Sketch.simple(
```

Comment:
This test description mentions `setup()`, but it doesn't provide a `setup()`. I think you're testing against the default `setup()` behavior, but that can be confusing for new developers. If you want to show that `clear()` overrides the `setup()` color, then it would be a good idea to explicitly set the background in `setup()`.